src/components/SectionForm/SectionForm.tsx

```diff
@@ -29,20 +29,30 @@ const submitTopics = (event:React.MouseEvent): void => {
 const sectionCheckboxes = topics.map(topic => {
   const key = Math.random();
   return (
-    <div className='article-section'>
-      <label>{topic}</label>
+    <span className="article-section">
       <input className="section" id={topic.toLowerCase()} type="checkbox" checked={selectedTopics.includes(topic) ? true : false} key={key} value={topic.toLowerCase()} onChange={() => handleChange(topic)}/>
-    </div>
+      <label className="topic-label" onClick={() => handleChange(topic)}>{topic}</label>
+    </span>
   )
 })
 
   return (
-    <form>
-    { sectionCheckboxes }
-      <button onClick={e => submitTopics(e)}>
-        Filter
-      </button>
-    </form>
+    <div className="filter-container">
+      <form className="filter-options hidden">
+        <div className="checkbox-container-1 checkbox-container">
+          { sectionCheckboxes[0] } { sectionCheckboxes[1] } { sectionCheckboxes[2] } { sectionCheckboxes[3] } { sectionCheckboxes[4] } { sectionCheckboxes[5] }
+        </div>
+        <div className="checkbox-container-2 checkbox-container">
+          { sectionCheckboxes[6] } { sectionCheckboxes[7] } { sectionCheckboxes[8] } { sectionCheckboxes[9] } { sectionCheckboxes[10] } { sectionCheckboxes[11] }
+        </div>
+        <div className="checkbox-container-3 checkbox-container">
+          { sectionCheckboxes[12] } { sectionCheckboxes[13] } { sectionCheckboxes[14] } { sectionCheckboxes[15] } { sectionCheckboxes[16] } { sectionCheckboxes[17] }
```

Comment:
This seems like we're hard-coding the number of topics. It might work for now, but we'll eventually want to switch to a dynamic filter section in which only topics that cover at least one article will be shown. 

If we fetch the top stories from the NYT, and there aren't any 'sports' articles, we probably shouldn't have a 'sports' filter. It's frustrating to click a filter option only to be told there are no articles that match that filter. As a user, I'd rather not see the option to filter by 'sports' in the first place.